docs/src/main/tut/GettingStarted.md

```diff
@@ -1,7 +1,7 @@
 # Getting started
```

Comment:
Very cool stuff @imarios, just a minor typo comment but aside from that it looks really great. Thank you !